src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,39 @@
+package controller;
+
+import entities.Hotel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Controller<T> {
```

Comment:
1. Why is this controller parameterized
2. You need one controller. Call it more specific, e.g., HotelController